WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -270,11 +292,27 @@ private extension ImageAndTitleAndTextTableViewCell {
     }
 
     func configureContentStackView() {
-        contentStackView.alignment = .center
+        contentStackView.alignment = .firstBaseline
         contentStackView.spacing = 16
     }
 
     func configureTitleAndTextStackView() {
         titleAndTextStackView.spacing = 2
     }
 }
+
+// MARK: Accessibility
+//
+private extension ImageAndTitleAndTextTableViewCell {
+    func applyAccessibilityChanges(contentSizeCategory: UIContentSizeCategory) {
+        adjustContentStackViewAxis(contentSizeCategory: contentSizeCategory)
+    }
+
+    /// Changes the image view width according to the base image dimension.
+    func adjustContentStackViewAxis(contentSizeCategory: UIContentSizeCategory) {
+        let isVerticalStack = contentSizeCategory >= .accessibilityMedium
+        contentStackView.axis = isVerticalStack ? .vertical: .horizontal
+        contentStackView.alignment = isVerticalStack ? .leading: .firstBaseline
+        contentStackView.spacing = isVerticalStack ? 5: 16
```

Comment:
Is there a reason why we're not using a multiple of `4`? Just curious. It can be really tricky to adjust pixels/points so I understand why we have to do this sometimes.

Reply:
Is a multiple of 4 a design preference documented somewhere? I tried 16px (too apart), 0px (too close), then 5px looks pretty good to me - didn't really think of the base multitude 😅 

Since G already did design review, let's go with this and we can always update the spacing later 🙂

Reply:
I was wrong. It's not `4` but `8`. 😄 From https://woomobiledesignhandbook.wordpress.com/layout/:

> All layouts are based on a 8pt grid, with a preference for 16pt as the default reference unit (i.e. spacing, gaps, padding).

Yup. I was just curious here and it was a non-blocking question. 🙂